bundle/camunda-saas-bundle/src/main/java/io/camunda/connector/runtime/saas/security/SecurityConfiguration.java

```diff
@@ -57,7 +57,10 @@ public WebSecurityCustomizer webSecurityCustomizer() {
   public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
     http.csrf(csrf -> csrf.ignoringRequestMatchers("/inbound/**"))
         .authorizeHttpRequests(
-            auth -> auth.requestMatchers("/inbound").hasAuthority("SCOPE_inbound:read"))
+            auth -> {
+              auth.requestMatchers("/inbound").hasAuthority("SCOPE_inbound:read");
+              auth.requestMatchers("/tenants/**").hasAuthority("SCOPE_inbound:read");
```

Comment:
This can be a single line: `auth.requestMatchers("/inbound", "/tenants/**").hasAuthority("SCOPE_inbound:read");`